### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by francescofuggitti**
This PR addresses issues #19 and #24. It also solves a bug in the appereance of the bibbase publication list.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -6,6 +6,8 @@
   <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
   <!-- <link rel="stylesheet" href="/css/main.css"> -->
   <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
+  <!-- <link rel="stylesheet" type="text/css"
+    href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" /> -->
```
Comment (marcofavorito) [inline _includes/head.html]: remove?

File: _pages/publications.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ permalink: /publications/
 
 # Publications
 
-**WARNING**: the website is under construction. The information
-might be inaccurate or wrong.
+<!-- **WARNING**: the website is under construction. The information
```
Comment (marcofavorito) [inline _pages/publications.md]: remove?

File: _includes/head.html (hunk 1 above)
Comment (francescofuggitti) [reply _includes/head.html]: I will keep it for now. It could be useful for solving bibbase css conflicts

File: _pages/publications.md (hunk 2 above)
Comment (francescofuggitti) [reply _pages/publications.md]: Good! Thanks

**Pull request merged**